ofl/notoseriftc/METADATA.pb

```diff
@@ -10,7 +10,7 @@ fonts {
   filename: "NotoSerifTC[wght].ttf"
   post_script_name: "NotoSerifTC-ExtraLight"
   full_name: "Noto Serif TC"
```

Comment:
@simoncozens The full font name should indeed contain the the style name. I overlooked that before. It is the postcript name that we want the same as before to not disturb Adobe.